src/math/Euler.js

```diff
@@ -280,10 +280,20 @@ THREE.Euler.prototype = {
 
 	},
 
-	toArray: function () {
+	toArray: function( array, offset, withoutOrder ) {
 
-		return [ this._x, this._y, this._z, this._order ];
+		if ( array === undefined ) array = [];
+		if ( offset === undefined ) offset = 0;
 
+		array[ offset + 0 ] = this._x;
+		array[ offset + 1 ] = this._y;
+		array[ offset + 2 ] = this._z;
+
+		if( !withoutOrder ) {
```

Comment:
I prefer to avoid double-negatives. Instead, do something like this:

    if( withOrder === true ) {

Perhaps there is a better API. If not, I'd recommend this change.